core/container/pool/pool.odin

```diff
@@ -0,0 +1,96 @@
+package container_pool
+
+import "core:fmt"
+
+// Single bit to mark a slot as active
+PoolFlag :: bit_set[0..<64;u64]
```

Comment:
The types should be named using Ada case, that is words are separated by an underscore and each word starts with a capital letter: `Pool_Flag`.